core/utils/promotions/validate-promotion.php

```diff
@@ -16,7 +16,8 @@ public static function domain_is_on_elementor_dot_com( $url ) {
 
 		$domain_segments = explode( '.', $domain );
 
-		if ( ! count( $domain_segments ) >= 2 ) {
+		$is_localhost = ( 1 === count( $domain_segments ) );
```

Comment:
When localhost is exploded in line 17 there's an undefined offset in line 24 because there is just one item in the $domain_segments array.